docs/console/docs/001-cr-microfrontend.md

```diff
@@ -0,0 +1,50 @@
+---
+title: MicroFrontend
+type: Custom Resource
+---
+
+The `microfrontend.ui.kyma-project.io` Custom Resource Definition (CRD) is a detailed description of the kind of data and the format used to extend the Kyma Console. It allows to extend the Console for the specific Namespace. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd microfrontends.ui.kyma-project.io -o yaml
+```
+
+## Sample Custom Resource
+
+This is a sample CR that extends the Console. 
+
+```
+apiVersion: ui.kyma-project.io/v1alpha1
+kind: MicroFrontend
+metadata:
+  name: sample-microfrontend
+  namespace: production
+spec:
+  version: 0.0.1
+  category: Sample Category
+  viewBaseUrl: https://sample-microfrontend-url.com
+  navigationNodes:
+    - label: Sample List
+      navigationPath: items
+      viewUrl: /
+    - label: Details
+      navigationPath: items/:id
+      showInNavigation: false
+      viewUrl: /:id
+```
+
+This table lists all the possible parameters of a given resource together with their descriptions:
+
+
+| Field   |      Mandatory?      |  Description |
+|:----------:|:-------------:|:------|
+| **metadata.name** | **YES** | Specifies the name of the Microfrontend. |
+| **metadata.namespace** | **YES** | Specifies the target namespace (environment) for the Microfrontend. |
+| **spec.version** | **NO** | Specifies the version of the Microfrontend. |
+| **spec.category** | **NO** | Defines the Category name under which the Microfrontend appears in the navigation. |
```

Comment:
`Specifies the category name under which the micro front-end appears in the navigation. `